modules/app.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-plusplus */
```

Comment:
- [ ] It looks like you have hard coded the date, while it is supposed to be delivered from the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) package. This is still in the **project requirements so please head back and see how to get the Date object from the Luxon package and display it in your page dynamically 😉**